src/common/ContractsBase.ts

```diff
@@ -34,23 +35,23 @@ export default class ContractsBase {
     return new web3.eth.Contract(ChildERC721Artifact.abi, token)
   }
 
+  public getChildMaticContract(childMaticAddress: address) {
+    return new this.web3Client.web3.eth.Contract(ChildMaticArtifact.abi, childMaticAddress)
```

Comment:
`childMaticAddress` is fixed. Can we make that a constant and use it here?

Reply:
actually, I required token address if the user wants raw data, so passing it as param
So basically It is declared const from where this function is called